### PR TITLE
Integrate Policy Check into Bump Version Tool

### LINKED ISSUE #5409: Write and Integrate Script into release that replaces assert messages with short codes
In order to reduce bundle size we should write a script that parses all out code, looks for a asserts, replaces the message with a short unique code, and puts the message in a comment.

### PR DESCRIPTION
**Pull request by anthony-murphy**
Integrate policy check into the bump version tool. This allows use to prepare the branch for release, by running things like assert short code replacement before release.

### CODE REVIEW

File: tools/build-tools/src/bumpVersion/bumpVersionCli.ts [hunk 1]
```diff
@@ -265,6 +265,15 @@ async function main() {
             fatal(`Local repo is dirty\n${status}`);
         }
 
+        console.log("Running Policy Check with Resolution(fix)");
+        await exec(`node ${__dirname}\\..\\repoPolicyCheck\\repoPolicyCheck.js -r -q`,resolvedRoot,"policy-check:fix failed");
+
```
Comment (anthony-murphy) [inline tools/build-tools/src/bumpVersion/bumpVersionCli.ts:270]: this is probably too aggressive, probably only want this for branch, release and bump

Comment (curtisman) [reply tools/build-tools/src/bumpVersion/bumpVersionCli.ts]: Yep... please scope this to only the commands that needs it.

Comment (curtisman) [reply tools/build-tools/src/bumpVersion/bumpVersionCli.ts]: And add comment about why and what this is try to accomplish.

File: tools/build-tools/src/bumpVersion/releaseVersion.ts [hunk 2]
```diff
@@ -25,6 +25,12 @@ export function getPackageShortName(pkgName: string) {
  * If --commit or --release is specified, the bumpped version changes will be committed and a release branch will be created
  */
 export async function releaseVersion(context: Context, releaseName: string, updateLock: boolean, releaseVersion?: VersionBumpType) {
+
+    // run policy check before creating release branch.
```
Comment (anthony-murphy) [inline tools/build-tools/src/bumpVersion/releaseVersion.ts:29]:
```suggestion
    // run policy check before releasing a version.
```

**Comment by anthony-murphy** [comment]
ping @curtisman

**Review by curtisman — APPROVED** [review]
:shipit:

**Pull request merged**